src/termit.h

```diff
@@ -21,7 +21,7 @@
 #include <config.h>
 
 #include <gtk/gtk.h>
-#include <vte/vte.h>
+#include <vte-2.91/vte/vte.h>
```

Comment:
It is not good "fix"
The path may be changed in future.
We should fellow the value from pkg-config
https://github.com/nonstop/termit/pull/131